webdriver-spec.html

```diff
@@ -4882,59 +4903,46 @@
   if it is a <a>success</a>.
   Otherwise return <var>element result</var>.
 
- <li><p>Run the substeps of the first step that case-insensitively
-  matches the <var>element</var>'s tag name:
-
-  <dl class=switch>
-   <dt><code>"input"</code>
-   <dd><p>If the <a><code>type</code> attribute</a>
-    is <code>"file"</code> return an <a>error</a> with <a>error
-    code</a> <a>invalid argument</a>. Otherwise,
-    let <var>container</var> equal <var>element</var>.
-
-   <dt><code>"option"</code>
-   <dd><p>Let <var>container</var> equal the <a><code>datalist</code>
-    element</a> or <a><code>select</code> element</a> of
-    which <var>element</var> is a child. If <var>container</var> is
-    undefined, return <a>error</a> with <a>error code</a> <a>invalid
-    argument</a>.
-
-   <dt>Otherwise
-   <dd><p>Let <var>container</var> equal <var>element</var>.
-  </dl>
-  </li>
+ <li><p>If the <var>element</var> is
+  an <a><code>input</code> element</a> in the <a>file upload state</a>
+  or it does not have a <a>container</a> <a>element</a>,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
- <li><p><a>Scroll into view</a> the <var>container</var>.
+ <li><p><a>Scroll into view</a>
+  the <var><a>element</a></var>’s <a>container</a> <a>element</a>.
```

Comment:
Second instance refers to the concept element, and `<var>` is used for the first.